include/gmock-win32.h

```diff
@@ -808,18 +808,31 @@ struct mock_module_##func \
 #define MOCK_MODULE_FUNC_OVERLOAD(name, ...) MOCK_MODULE_UNITE(MOCK_MODULE_OVERLOAD(name, MOCK_MODULE_NBARG(__VA_ARGS__)), (__VA_ARGS__))
 #define MOCK_MODULE_FUNC(r, m, ...) MOCK_MODULE_OVERLOAD(MOCK_MODULE_FUNC, MOCK_MODULE_NBARG(__VA_ARGS__))##(m, r(__VA_ARGS__))
 
-#define MOCK_STDCALL_FUNC(r, m, ...) MOCK_MODULE_OVERLOAD(MOCK_MODULE_FUNC, MOCK_MODULE_NBARG(__VA_ARGS__)##_STDCALL_CONV)##(m, r(__VA_ARGS__))
+#define MOCK_STDCALL_FUNC(r, m, ...) \
+	MOCK_MODULE_OVERLOAD(MOCK_MODULE_FUNC, MOCK_MODULE_NBARG(__VA_ARGS__)##_STDCALL_CONV)##(m, r(__VA_ARGS__)) \
+	_Pragma( "optimize( \"\", off )" ) \
```

Comment:
"I checked the pragma optimization `_Pragma("optimize("", off)")` on my MSVC 2015, and it doesn't work. Can we create a macro to support it? For example:
```cpp
#if defined(_MSC_VER) && _MSC_VER >= 1910
    #define MOCK_MODULE_OPTIMIZATION_ENABLE     _Pragma("optimize(\"\", on)")
    #define MOCK_MODULE_OPTIMIZATION_DISABLE    _Pragma("optimize(\"\", off)")
#else
    #define MOCK_MODULE_OPTIMIZATION_ENABLE     __pragma(optimize("", on))
    #define MOCK_MODULE_OPTIMIZATION_DISABLE    __pragma(optimize("", off))
#endif
```

Reply:
Of course, I'll do it now.
I hope the building was like this:
`x86/cl.exe /Gy /GL /Ox`
`x86/link.exe /LTCG:incremental`